tests/pages/user-page.js

```diff
@@ -11,5 +11,7 @@ export default createPage({
     hover: triggerable('mouseenter'),
   },
 
+  courseLabels: collection('[data-test-course-name]'),
```

Comment:
Let's try to map these to component names. In this case, it'd be `courseProgressListItems`. That way, in the future if we want to test other properties (like progress bar width etc.), it's easy to add those properties to this page object.